packages/address/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "dependencies": {
-    "@ethersproject/bignumber": "^5.5.0",
-    "@ethersproject/bytes": "^5.5.0",
-    "@ethersproject/keccak256": "^5.5.0",
-    "@ethersproject/logger": "^5.5.0",
-    "@ethersproject/rlp": "^5.5.0"
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@ethersproject/keccak256": "5.5.0",
+    "@hethers/logger": "1.0.0",
+    "@ethersproject/rlp": "5.5.0"
```

Comment:
I think that we do not need the `rlp` package

Reply:
`rlp` package is removed